vision/genesis_principles.md

```diff
@@ -0,0 +1,14 @@
+# ₿trust Genesis Principles
+₿trust’s Genesis Principles are an immutable set of principles governing its operation and outlining its vision. They exist to guide the board and representatives of the trust, and to act as an accountability safeguard against bad actors.  
+
+Board members will take ownership of this vision, and act in good faith to protect the interests and integrity of the organization. The specific implementation of these principles is the prerogative of the board and its selected representatives.
+
+1. Bitcoin will provide a monetary system that enhances financial sovereignty for all. ₿trust will support Bitcoin exclusively. In the event of a hard fork it will follow the chain endorsed by the user majority.
+
+2. ₿trust will maintain the highest levels of integrity. To that end, ₿trust does not exist to make a profit for itself or its representatives. The board will be mindful to prevent self-dealing within the organization and representatives will declare conflicts of interest as they arise.
```

Comment:
```suggestion
2. ₿trust will maintain the highest levels of integrity. To that end, ₿trust does not exist to make a profit for itself or its representatives. The board will be mindful to prevent self-dealing within the organization, and representatives will declare conflicts of interest as they arise.

```